src/cargo/ops/cargo_new.rs

```diff
@@ -128,8 +125,6 @@ impl NewOptions {
 
 #[derive(Deserialize)]
 struct CargoNewConfig {
-    name: Option<String>,
-    email: Option<String>,
```

Comment:
Can these be kept, so that Cargo doesn't immediately start issuing a warning about unknown fields?  They can be removed in the future.  This can improve the experience for people who use different rust toolchain versions.

Reply:
Did this by adding an `#[allow(dead_code)]` and a `#[deprecated]` (to make it clear what it's doing there). Not sure if there is a better way of doing this.